box-share-sdk/src/main/java/com/box/androidsdk/share/vm/InviteCollaboratorsVM.java

```diff
@@ -0,0 +1,274 @@
+    }
+
+    /**
+     * Returns a LiveData which holds a data wrapper that contains a list of invitees that can be invited and a string resource code.
+     * @return a LiveData which holds a data wrapper that contains a list of invitees that can be invited and a string resource code
+     */
+    public LiveData<DataWrapper<BoxIteratorInvitees>> getInvitees() {
+        return mInvitees;
+    }
+
+    /**
+     * Helper method for transforming BoxResponse to UI Model for fetchRoleApi
+     * @param response the response to transform on
+     * @return the transformed data
+     */
+    private static DataWrapper<BoxCollaborationItem> createFetchRoleItemData(BoxResponse<BoxCollaborationItem> response) {
+        final DataWrapper<BoxCollaborationItem> data = new DataWrapper<BoxCollaborationItem>();
+        if (response.isSuccess()) {
+            BoxCollaborationItem collaborationItem = response.getResult();
+            data.success(collaborationItem);
+        } else {
+            data.failure(R.string.box_sharesdk_network_error);
+        }
+        return data;
+    }
+    /**
+     * Helper method for transforming BoxResponse to UI Model for getInviteesApi
+     * @param response the response to transform
+     * @return the transformed model
+     */
+    private static DataWrapper<BoxIteratorInvitees> createGetInviteesItemData(BoxResponse<BoxIteratorInvitees> response) {
+        final DataWrapper<BoxIteratorInvitees> data = new DataWrapper<BoxIteratorInvitees>();
+        if (response.isSuccess()) {
+            final BoxIteratorInvitees invitees = response.getResult();
+            data.success(invitees);
+        } else {
+            BoxException boxException = (BoxException) response.getException();
+            int responseCode = boxException.getResponseCode();
+            int errorStrCode = R.string.box_sharesdk_generic_error;
+            if (responseCode == HttpURLConnection.HTTP_FORBIDDEN) {
+                errorStrCode = R.string.box_sharesdk_insufficient_permissions;
+            } else if (boxException.getErrorType() == BoxException.ErrorType.NETWORK_ERROR) {
+                errorStrCode = R.string.box_sharesdk_network_error;
+            }
+            data.failure(errorStrCode);
+        }
+        return data;
+    }
+    /**
+     * Helper method for transforming BoxResponse to UI Model for addCollabsApi
+     * @param response the response to transform
+     * @return the transformed model
+     */
+    private static InviteCollaboratorsDataWrapper createAddCollabsItemData(BoxResponse<BoxResponseBatch> response) {
+        return handleCollaboratorsInvited(response.getResult());
+    }
+
+    /**
+     * Helper method for helping with transformation of BoxBatchResponse to UI Model
+     * @param responses the batch response to transform
+     * @return the transformed model
+     */
+    private static InviteCollaboratorsDataWrapper handleCollaboratorsInvited(BoxResponseBatch responses) {
```

Comment:
I'd name this based on what it is expected to return. For e.g. getInvitedCollaborators

Reply:
I will make the change.